Tests/Integration/ClassCIntegrationTests.cs

```diff
@@ -50,7 +66,7 @@ public async Task When_ABP_Sends_Upstream_Followed_By_DirectMethod_Should_Send_U
 
             var twin = simDevice.CreateABPTwin(reportedProperties: new Dictionary<string, object>
                 {
-                    { TwinProperty.Region, LoRaRegionType.EU868.ToString() }
+                    { TwinProperty.Region, regionType.ToString() }
```

Comment:
On L98, where we use `CreateWaitableRequest`, we always create the request for region EU868. I think we should use the injected lora region type consistently